src/odemis/driver/actuator.py

```diff
@@ -1302,7 +1300,7 @@ def _doReference(self, axes):
         # If we just did homing and ended up to an unsupported position, move to
         # the nearest supported position
         cp = self._child.position.value[self._caxis]
-        if cp not in self._positions:
+        if (cp not in self._positions):
```

Comment:
That was fine. No need for a revert.

Reply:
ah I think that was when reverted by loading changes from the disc. Then it reverts everything in the file back...need to check this next time separately.